assets/models/easter_eggs/SporeCell.tscn

```diff
@@ -0,0 +1,505 @@
+"blend_shape_data": [  ],
+"format": 2194903,
+"index_count": 1296,
+"material": ExtResource( 2 ),
+"primitive": 4,
+"skeleton_aabb": [ AABB( -1.46845, -1.40214, -2.25429, 2.93691, 2.80427, 2.30923 ), AABB( -1.46845, -1.40214, -2.25429, 2.93691, 2.80427, 3.14247 ), AABB( -1.24065, -1.36023, -0.88309, 2.48131, 2.65776, 3.04678 ), AABB( -0.472767, -0.412523, 1.91071, 0.472777, 0.825046, 0.252986 ), AABB( 0, 0, 0, -1, -1, -1 ), AABB( 0, -0.412523, 1.91071, 0.472767, 0.825046, 0.252976 ), AABB( 0, 0, 0, -1, -1, -1 ), AABB( -1.74808, -1.40214, -2.09403, 2.34498, 2.80427, 4.08859 ), AABB( -2.09104, -1.0902, -1.62873, 1.40571, 2.45044, 2.71561 ), AABB( -2.61336, -0.36634, 0.315248, 1.00556, 0.670542, 0.842363 ), AABB( 0, 0, 0, -1, -1, -1 ), AABB( -0.455002, -1.40214, -2.00178, 2.20308, 2.80427, 3.99635 ), AABB( 0.685331, -1.36023, -0.993029, 1.40571, 2.72047, 2.07991 ), AABB( 1.6078, -0.36634, 0.315248, 1.00556, 0.670541, 0.842363 ), AABB( 0, 0, 0, -1, -1, -1 ), AABB( -0.907669, -1.23504, -2.28006, 2.14519, 2.47007, 0.670185 ) ],
```

Comment:
CI is complaining about this embedded data, basically the data needs to be in a separate .mesh or a .glb file for Git LFS to work.